zabbix/check.py

```diff
@@ -0,0 +1,334 @@
+"""
+    StackState.
+    Zabbix host topology and problem integration
+
+    Current limitations:
+    - Unsupported Zabbix Problem acks
+    - Unsupported Zabbix version 4+ severity upgrades, currently the trigger's severity is used for compatibility
+"""
+
+import requests
+import logging
+import time
+
+from checks import AgentCheck, CheckException
+
+
+class ZabbixHost:
+    def __init__(self, hostid, host, name, host_groups):
+        assert(type(host_groups == list))
```

Comment:
is it just to make sure that host_groups should be in the list?

Reply:
Yeah, just want a bit of type safety. There may be a more elegant way or it may be superfluous.